_pages/profile/[id]/submission-details-card/index.js

```diff
@@ -133,8 +136,36 @@ export default function SubmissionDetailsCard({
     submissionDetailsCardFragments.submission,
     submission
   ));
-
   const [accounts] = useWeb3("eth", "getAccounts");
+  const { props } = useQuery(
+    appQuery,
+    {
+      id: accounts?.[0]?.toLowerCase(),
+      contributor: accounts?.[0]?.toLowerCase(),
+    },
+    { skip: !accounts?.[0] }
+  );
+
+  const disputedVouches = props?.submission?.vouchees.filter(
+    (vouchee) => vouchee?.disputed === true
+  );
+
+  let vouchingTimeout = false;
+  if (disputedVouches?.length > 0) {
+    const latestChallengedVouchee = disputedVouches?.sort((a, b) =>
+      a.requests[0].challenges[0].creationTime <
+      b.requests[0].challenges[0].creationTime
+        ? 1
+        : -1
+    );
+
+    const challengeTimestamp =
+      latestChallengedVouchee[0].requests[0].challenges[0].creationTime * 1000;
+    const timestamp = challengeTimestamp + 5200000000 > Date.now();
```

Comment:
How did you calculate the number `5200000000`?

Reply:
It’s the approximate amount of seconds that there are in 60 days.

Reply:
@bilinkis Then I suggest you replace the number with `60*24*60*60` (for increased readability):

```suggestion
    const timestamp = challengeTimestamp + 60*24*60*60 > Date.now();
```

Reply:
It should be also multiplied by 1000 given that I the output of Date.now is in milliseconds. But it's a good catch. I'll commit that now.